com.unity.renderstreaming/Documentation~/data-streaming.md

```diff
@@ -0,0 +1,57 @@
+# Data Streaming
+
+Data streaming is using [`RTCDataChannel`](https://docs.unity3d.com/Packages/com.unity.webrtc@2.4/manual/datachannel.html) of the WebRTC package internally.
```

Comment:
```suggestion
Data streaming is using [`RTCDataChannel`](https://docs.unity3d.com/Packages/com.unity.webrtc@latest/manual/datachannel.html) of the WebRTC package internally.
```

Reply:
Unfortunately, `@latest` is not worked correctly.

Work : https://docs.unity3d.com/Packages/com.unity.webrtc@latest
Not work : https://docs.unity3d.com/Packages/com.unity.webrtc@latest/manual/datachannel.html